src/App.tsx

```diff
@@ -30,21 +36,9 @@ function App() {
           }}
         />
       </div>
-
       <div className="cards-container">
```

Comment:
Need to put only `container`
```suggestion
      <div className="container">
```